docs/products/flink/index.rst

```diff
@@ -1,10 +1,10 @@
-Apache Flink :badge:`beta,cls=badge-secondary text-black badge-pill`
-====================================================================
+Apache Flink® :badge:`beta,cls=badge-secondary text-black badge-pill`
+=====================================================================
 
-What is Aiven for Apache Flink?
--------------------------------
+What is Aiven for Apache Flink®?
+--------------------------------
 
-Aiven for Apache Flink beta is powered by the open-source framework Apache Flink, a **distributed processing engine for stateful computations over data streams**. It enables you to easily get started with real-time stream processing using SQL.
+Aiven for Apache Flink® beta is powered by the open-source framework Apache Flink®, a **distributed processing engine for stateful cs over data streams**. It enables you to easily get started with real-time stream processing using SQL.
```

Comment:
This looks like a mistake? cs instead of computations?

Reply:
Agreed. Probably an editor miskey. Thanks.